extensions/wikia/ArticleAsJson/ArticleAsJson.class.php

```diff
@@ -8,7 +8,6 @@ class ArticleAsJson extends WikiaService {
 	];
 
 	const CACHE_VERSION = '0.0.3';
-	const CACHE_VERSION_FOR_SEO_FRIENDLY_IMAGES = 1; // for $wgEnableSeoFriendlyImagesForMobile
```

Comment:
You're right, I think it's safer to use `1` as not touching is always safer :wink: